etna/lib/etna/spec/vcr.rb

```diff
@@ -1,9 +1,27 @@
+require 'cgi'
 require 'webmock/rspec'
 require 'vcr'
 require 'openssl'
 require 'digest/sha2'
 require 'base64'
 
+def clean_query(json_or_string)
+  if json_or_string.is_a?(Hash) && json_or_string.include?('upload_path')
```

Comment:
upload paths have non deterministic urls thanks to Nonce, so in test handling I created this helped to make it easier to clean them out from those embedded urls by replacing them with the consistent string 'etna-signature'.  This is opt-in helper function and not default behavior.